hibernate-search-plugins/org.jboss.tools.hibernate.search.runtime.v_4_0/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Just wondering. Is it normal for the jboss-tools project to commit the Eclipse project files ?

I'm honestly intriguided, as we strictly avoid this in Hibernate projects - but I'm not sure why :)